website/blog/2022/07/05/use-dataant-to-monitor-apisix.md

```diff
@@ -0,0 +1,92 @@
+
+## Introduction
+
+![principle](https://static.apiseven.com/2022/blog/0705/1.svg)
+
+1. Collection configuration
+
+  The DataAnt Agent will first initialize and register the collector through the APISIX item in `config.yaml`. The same Agent can register multiple collectors. After the collector collects the indicators exposed by APISIX, it encrypts the indicator data and uploads it to DataAnt Cloud.
+
+2. Data visualization
+
+  After DataAnt Cloud receives the data, the data will be stored in the time series database after preliminary monitoring information supplementation and processing, and then APISIX can be monitored in real time through DataAnt's Dashboard.
+
+3. Alarm notification
+
+  The data will also be distributed to the alarm matching processing through the message, and then the notification aggregation will be carried out. Finally, the alarm will be sent through the configured notification method, that is, the abnormal situation of APISIX can be received in real time.
+
+## Operation steps
+
+  1. First, you need to access [DataAnt Cloud](http://139.224.11.158), register an account and log in to the platform.
+
+  2. Obtain the Agent of DataAnt through the [download link](https://pan.baidu.com/s/1fabvSiDLDh8ZRTjpzINHLg?pwd=87d4). After the download is complete, upload it to the machine where APISIX is located and add execution permission to the Agent.
+
+  3. Create the configuration file `./config.yaml` required by DataAnt Agent in the current directory. The detailed configuration is as follows:
+
+  ```yaml
+  tenantId: 11
+  hostIp: 127.0.0.1
+  hostName: apisix
+  configs:
+  - uri: http://127.0.0.1:9091
+    type: apisix
+    asName: apisix_test
+  ```
+
+  4. Start the Agent using the following command.。
+
+  ```shell
+  ./agent
+  ```
+
+  After successful startup, the following data will be returned:
+
+   ```shell
+  2022/06/21 20:50:10 {"code":200,"msg":"请求成功","data":null}
+  2022/06/21 20:50:30 {"code":200,"msg":"请求成功","data":null}
+  2022-06-21 20:51:00:000        INFO        apisix/apisix.go:25        获取对应监控数据，数据长度1675
+  2022-06-21 20:51:00:000        INFO        prometheus/prometheusCollector.go:43        获取对应监控数据开始解析1675
+  2022-06-21 20:51:00:000        INFO        prometheus/prometheusCollector.go:43        获取对应监控数据完成解析 解析指标数量21
+  2022-06-21 20:51:00:000        INFO        collector/collector.go:82        apisix采集到数据数量21
+  2022-06-21 20:51:00:000        INFO        runtime/asm_amd64.s:1581        apisix_test9091:指标数:21
+  ```
+
```

Comment:
Content in the code block shows up in a wrong way.
<img width="807" alt="image" src="https://user-images.githubusercontent.com/36651058/178170718-ce24c691-35c8-4d36-917d-68295cf06691.png">